### DISCUSSION TITLE
CI with codecov coverage

### DISCUSSION THREAD

**Discussion by VietAnh1010** [joulev/ezkomment#183] [Questions]
![image](https://user-images.githubusercontent.com/91171052/178178743-c4d15e99-e82d-4161-bdab-9e922190dc82.png)

Must the coverage on other branches be greater than or equal to the coverage on `main` branch to pass codecov coverage check?

**Comment by joulev** [comment]
No, but the diff needs to be higher than -5%.

I think for your commit there was some problems uploading the results to Codecov. Try rerunning the workflow.

**Reply by VietAnh1010** [reply]
It still fails. Can you take a look at that?

**Reply by joulev** [reply]
hmm it's very weird, I'm not sure why either. Anyway that still means there are tests that need to be written.
